src/test/java/com/fasterxml/jackson/databind/deser/TestFloats.java

```diff
@@ -50,6 +54,28 @@ public void testArrayOfFloatPrimitives() throws Exception
         assertEquals("1.4E-45", Float.toString(floats[3])); //this assertion fails unless toString is used
     }
 
+    public void testBigArrayOfFloatPrimitives() throws Exception {
+        StringBuilder sb = new StringBuilder(1024);
+        ObjectMapper mapper = new ObjectMapper();
+        try (
+                InputStream stream = TestFloats.class.getResourceAsStream("/data/float-array.txt");
```

Comment:
Why not simply pass this `InputStream` to `ObjectMapper`? Or does that not exhibit the issue?

Reply:
made the requested change - does not affect the test